test/mocha/core/curlify.js

```diff
@@ -194,6 +194,54 @@ describe("curlify", function() {
         expect(curlified).toEqual("curl -X POST \"http://example.com\" -H  \"content-type: multipart/form-data\" -F \"id=123\" -F \"file=@file.txt;type=text/plain\"")
     })
 
+    it("should print a curl with formData and file, and trailing custom header", function () {
```

Comment:
Mocha allows us to structure tests into logical groups by using `context/specify` aliases.
```js
context('given request with multipart/form-data content type and file', function () {
  context('and trailing custom header', function() {
    specify('should print a proper curl', function () {
      ...test..
    });
  });

  context('and leading custom header', function() {
    specify('should print a proper curl', function () {
      ...test..
    });
  });
});